client/js/widgets/card.js

```diff
@@ -62,6 +62,9 @@ class Card extends Widget {
     for(const face of faceTemplates) {
       const faceDiv = document.createElement('div');
 
+      if (face.css !== undefined) {
```

Comment:
Most of the code omits braces for one line and has no space after `if`s. It's not important though.